File: src/URI/URI_1044.java
package URI;
import java.util.Scanner;

public class URI_1044 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int a = sc.nextInt();
		int b = sc.nextInt();			
		
		if(a % b == 0 || b % a == 0) {
			System.out.println("Sao Multiplos");
		}
		else {
			System.out.println("Nao sao Multiplos");
		}		
		sc.close();
	}

}

File: kotlin/URI_1044.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val a = sc.nextInt()
    val b = sc.nextInt()

    if(a % b == 0 || b % a == 0) {
        println("Sao Multiplos");
    }
    else {
        println("Nao sao Multiplos");
    }
}
